src/ai/backend/manager/registry.py

```diff
@@ -457,6 +458,10 @@ async def create_session(
                     ],
                 )
             requested_image_ref = image_row.image_ref
+            if (
+                _owner_uuid := image_row.labels.get("ai.backend.image-owner")
+            ) and _owner_uuid != str(user_scope.user_uuid):
```

Comment:
We plan to expand this feature to support per-project custom images.
Let's use quota-scope-like prefixes to distinguish whether the given owner ID is a user or a project.